lib/browser/browser.ts

```diff
@@ -210,48 +208,55 @@ class OktaAuthBrowser extends OktaAuthBase implements OktaAuth, SignoutAPI {
     this.authStateManager = new AuthStateManager(this);
   }
 
+  // TODO: deprecate in 5.0
```

Comment:
Is this deprecated now and removed in 5.0, or not deprecated until 5.0, and removed after that?  

The comment and message should be a little more clear